Library/Formula/rtv.rb

```diff
@@ -0,0 +1,62 @@
+class Rtv < Formula
+  desc "Command-line Reddit client"
+  homepage "https://github.com/michael-lazar/rtv"
+  url "https://github.com/michael-lazar/rtv/archive/v1.8.0.tar.gz"
+  sha256 "90828ad016d669dfc162ff958124700abcce7f2cf1aedcf5be90fe97f4552764"
+
+  depends_on :python3
+
+  resource "decorator" do
+    url "https://pypi.python.org/packages/source/d/decorator/decorator-4.0.6.tar.gz"
+    sha256 "1c6254597777fd003da2e8fb503c3dbf3d9e8f8d55f054709c0e65be3467209c"
+  end
+
+  resource "update_checker" do
```

Comment:
Alright, great. As long as the tool isn't trying to auto-update itself that's fine.